src/DDTrace/Configuration/WildcardToRegex.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace DDTrace\Configuration;
```

Comment:
I love this class, I wonder why it is in the `Configuration` namespace though? :)

Reply:
Very good point. Naming is hard. 😄

What do you think about adding an `Obfuscation` namespace? We could also move [the existing obfuscation stuff](https://github.com/DataDog/dd-trace-php/blob/master/src/DDTrace/Obfuscation.php) to that namespace in a separate PR. And possibly organize that code a bit better. What do you think? :)